config.go

```diff
@@ -242,6 +242,10 @@ type Config struct {
 
 	KeysendHoldTime time.Duration `long:"keysend-hold-time" description:"If non-zero, keysend payments are accepted but not immediately settled. If the payment isn't settled manually after the specified time, it is canceled automatically. [experimental]"`
 
+	GcSettledInvoices bool `long:"gc-settled-invoices" description:"If true, we'll attempt to garbage collect settled invoices upon start."`
```

Comment:
I'm not sure users really ever want to delete _settled_ invoices, as this is financial data.

Reply:
Yeah, I wasn't sure if we have users who want to do this. Removed for now.